kits/multisig/package.json

```diff
@@ -18,7 +18,9 @@
     "coverage": "./node_modules/@aragon/test-helpers/run-coverage.sh",
     "console": "truffle dev",
     "ganache-cli:dev": "./node_modules/@aragon/test-helpers/ganache-cli.sh",
-    "ganache-cli:coverage": "SOLIDITY_COVERAGE=true npm run ganache-cli:dev"
+    "ganache-cli:coverage": "SOLIDITY_COVERAGE=true npm run ganache-cli:dev",
+    "publish:rpc": "aragon apm publish $(npm run deploy:kit:rpc | tail -n 1) --network rpc",
+    "deploy:kit:rpc": "aragon contracts compile --network rpc && aragon contracts exec --network rpc scripts/multisig-deploy.js"
```

Comment:
It seems really weird we need to use `--network rpc` for `aragon contracts compile` but I'm guessing the CLI requires it?

Reply:
Yup, agreed. It is a bug in the CLI which requires passing a network for that command.

Reply:
Will be fixed here: https://github.com/aragon/aragon-cli/pull/230/files#r227244994